src/i18n/locales/fr.json

```diff
@@ -41,7 +41,7 @@
   "components.pagination.next": "Suivant",
   "components.randomContributor.thankYou": "Merci {contributor} d'être un contributeur Node.js avec {amount}  de contributions.",
   "components.recentPosts.title": "Recent Posts",
-  "components.searchBar.placeholder": "Search",
+  "components.searchBar.placeholder": "Recherche",
```

Comment:
Let's keep this one as "Search", otherwise the style breaks.

Reply:
OK